com.unity.visualeffectgraph/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [13.1.2] - 2021-11-08
+## [13.1.2] - 2021-11-05
```

Comment:
note: weird that VFX was already 13.1.2?

Reply:
I think Ludovic bumped it in this PR already https://github.com/Unity-Technologies/Graphics/pull/6238. Not sure if that was done knowing the previous srp2core PR was already made but it's nice :)